ui/raidboss/autoplay_helper.ts

```diff
@@ -1,20 +1,20 @@
 export default class AutoplayHelper {
-  static Check() {
+  static Check(): boolean {
```

Comment:
For future conversions, can you do one file at a time (unless it's so interrelated that it's hard to do them separately)?  This is fine for now.